components/common/Link.vue

```diff
@@ -33,9 +37,9 @@ export default {
   },
   computed: {
     type() {
-      if (this.to !== '') {
+      if (this.to) {
         return 'nuxt-link'
-      } else if (this.href !== '') {
+      } else if (this.href || this.mailto) {
```

Comment:
Since you have `null` default values is better to be explicit when comparing variables. Even if it is the same, I'd use `<var> !== null` to avoid unwanted comparisons.